dbt/models/overview.md

```diff
@@ -30,9 +30,9 @@ or table by clicking the teal button on the bottom right of this page. This
 lets you explore the upstream and downstream dependencies of any asset.
 Assets in the graph are color-coded by type:
 
-- [Sources](https://docs.getdbt.com/docs/build/sources) - Data loaded into the warehouse by scripts and other tools ![source](/data-architecture/assets/swatch-source.png)
-- [Models](https://docs.getdbt.com/docs/build/models) - Data transformation and assets created and managed by dbt ![model](/data-architecture/assets/swatch-model.png)
-- [Exposures](https://docs.getdbt.com/docs/build/exposures) - Downstream consumers of dbt assets e.g. [Open Data](https://datacatalog.cookcountyil.gov/browse?tags=cook+county+assessor) ![exposure](/data-architecture/assets/swatch-exposure.png)
+- [Sources](https://docs.getdbt.com/docs/build/sources) - Data loaded into the warehouse by scripts and other tools ![source](./assets/swatch-source.png)
```

Comment:
I _think_ these relative links should work fine, but it's difficult to test since we don't have deploy previews setup. If they break I can fix them with a fix follow-up commit.

Reply:
**[Praise]** Yup, I rendered and served the docs locally and can confirm that these links work!